### PR TITLE
[Version] Bump version to v0.6.2

### PR DESCRIPTION
**Pull request by HaiyiMei**
- [Unreal] Add lineart annotation
- [Unreal] Fix bug of `Annotator`
    - Fix the way of finding active playing sequence
    - Replace euler with quat when exporting ActorInfo
- [Unreal] Replace assert error with warning when adding FK motion. This allows you to add motions in an inaccurate way.
- Fix smplx_annotation accuracy issue
    - Add `transform_smpl_x` in `xrfeitoria/utils/anim/motion.py`

### CODE REVIEW

File: samples/blender/07_amass.py [hunk 1]
```diff
@@ -57,34 +59,47 @@ def main(background: bool = False):
...
         dump_humandata(motion, save_filepath=saved_humandata_file, meta_filepath=smpl_xl_meta_file)
+        refine_smpl_x(
+            smpl_x_filepath=saved_humandata_file,
+            meta_filepath=smpl_xl_meta_file,
+            offset_location=ConverterBlender.location_from_blender(location),
+            offset_rotation=ConverterBlender.rotation_from_blender(rotation),
+            replace_smpl_x_file=True,
+        )
 
-        # Modify the frame range to the length of the motion
```
Comment (wentww) [inline samples/blender/07_amass.py:79]: Why this code block is removed?

File: samples/blender/07_amass.py [hunk 2]
```diff
@@ -57,34 +59,47 @@ def main(background: bool = False):
...
-        frame_start, frame_end = xf_runner.utils.get_keys_range()
-        xf_runner.utils.set_frame_range(frame_start, frame_end)
```
Comment (wentww) [inline samples/blender/07_amass.py:81]: Why this code block is removed?

File: src/XRFeitoriaUnreal/Content/Python/sequence.py [hunk 3]
```diff
@@ -459,21 +459,22 @@ def add_fk_motion_to_binding(binding: unreal.SequencerBindingProxy, motion_data:
...
-            bone_name = f'{bone_name}_CONTROL'
-        assert bone_name in param_names, RuntimeError(f'bone name: {bone_name} not in param names: {param_names}')
+    # Note: assert bone_name is now achieved in function `xf_runner.SequenceUnreal._spawn_actor_in_engine`
+    # param_names = list(rig_section.get_parameter_names())
+    # for bone_name, bone_data in motion_data[0].items():
+    #     if 'curve' in bone_data.keys():
+    #         bone_name = f'{bone_name}_CURVE_CONTROL'
+    #     else:
+    #         bone_name = f'{bone_name}_CONTROL'
+    #     assert bone_name in param_names, RuntimeError(f'bone name: {bone_name} not in param names: {param_names}')
```
Comment (wentww) [inline src/XRFeitoriaUnreal/Content/Python/sequence.py:469]: Do you mean that the functionality of this code has been implemented in function `xf_runner.SequenceUnreal._spawn_actor_in_engine`? so whether this code should be removed?

File: xrfeitoria/renderer/renderer_unreal.py [hunk 4]
```diff
@@ -223,11 +223,11 @@ def convert_actor_infos(folder: Path) -> None:
...
                 with open(actor_info_file, 'rb') as f:
-                    dat = np.frombuffer(f.read(), np.float32).reshape(7)
+                    dat = np.frombuffer(f.read(), np.float32).reshape(8)
                 location.append(ConverterUnreal.location_from_ue(dat[:3]))
-                rotation.append(ConverterUnreal.rotation_from_ue(dat[3:6]))
-                stencil_value.append(int(dat[6]))
-                mask_color.append(unreal_functions.get_mask_color(int(dat[6])))
+                rotation.append(ConverterUnreal.quat_from_ue(dat[3:7]))
+                stencil_value.append(int(dat[7]))
+                mask_color.append(unreal_functions.get_mask_color(int(dat[7])))
```
Comment (wentww) [inline xrfeitoria/renderer/renderer_unreal.py:230]: Maybe it is better to use constants to document these indices and the number of annotations.

File: xrfeitoria/utils/anim/utils.py [hunk 5]
```diff
@@ -120,59 +120,138 @@ def dump_humandata(
     )
 
 
+def batch_transform_to_camera_frame(global_orient, transl, pelvis, extrinsic):
```
Comment (wentww) [inline xrfeitoria/utils/anim/utils.py:123]: Maybe this function should be renamed to remove the `camera` in the function name.

**Review by wentww — COMMENTED** [review]
add a comment

File: src/XRFeitoriaUnreal/Content/Python/sequence.py (hunk 3 above)
Comment (HaiyiMei) [reply src/XRFeitoriaUnreal/Content/Python/sequence.py]: > [Unreal] Replace assert error with warning when adding FK motion. This allows you to add motions in an inaccurate way.

To achieve this, the assert of errors must be abandoned. Instead, the warning will be performed in:

https://github.com/openxrlab/xrfeitoria/blob/d9c48ba3e142ecc6dcbfce61e982dd6ce7bd992b/xrfeitoria/sequence/sequence_unreal.py#L298-L326
https://github.com/openxrlab/xrfeitoria/blob/d9c48ba3e142ecc6dcbfce61e982dd6ce7bd992b/xrfeitoria/sequence/sequence_unreal.py#L171
https://github.com/openxrlab/xrfeitoria/blob/d9c48ba3e142ecc6dcbfce61e982dd6ce7bd992b/xrfeitoria/sequence/sequence_unreal.py#L214

File: xrfeitoria/renderer/renderer_unreal.py (hunk 4 above)
Comment (HaiyiMei) [reply xrfeitoria/renderer/renderer_unreal.py:230]: Yes, it's better to annotate these things somewhere, but it's not a trivial thing. It involves exporting data from C++ and reading data in Python. I'll come up with something reasonable in the future version.

File: xrfeitoria/utils/anim/utils.py (hunk 5 above)
Comment (HaiyiMei) [reply xrfeitoria/utils/anim/utils.py]: Solved in [`e6ee494` (#28)](https://github.com/openxrlab/xrfeitoria/pull/28/commits/e6ee49459ec673f979fef8f14e0bbebebdd8a453)

**Review by wentww — APPROVED** [review]
approve merge

**Pull request merged**